.ci/packaging.groovy

```diff
@@ -127,17 +109,9 @@ pipeline {
             expression { return env.IS_BRANCH_AVAILABLE == "true" }
           }
           steps {
-            googleStorageDownload(bucketUri: "gs://${JOB_GCS_BUCKET}/${URI_SUFFIX}/*",
-                                  credentialsId: "${JOB_GCS_CREDENTIALS}",
-                                  localDirectory: "${BASE_DIR}/build/distributions",
-                                  pathPrefix: env.PATH_PREFIX)
-            dir("${BASE_DIR}") {
-              dockerLogin(secret: env.DOCKER_SECRET, registry: env.DOCKER_REGISTRY)
-              script {
-                getVaultSecret.readSecretWrapper {
-                  sh(label: 'release-manager.sh', script: '.ci/scripts/release-manager.sh')
-                }
-              }
+            releaseManager(type: 'snapshot')
+            whenFalse(env.BRANCH_NAME.equals('main')) {
+              releaseManager(type: 'staging')
             }
```

Comment:
Run always `snapshot` for `main`/`8.1`/`8.2` branches and also `staging` for `8.1`/`8.2` branches